src/loaddb/load_db.c

```diff
@@ -50,6 +50,13 @@ static FILE *loaddb_log_file;
 int interrupt_query = false;
 bool load_interrupted = false;
 
+typedef struct t_mulitload_schema_file_info T_MULTILOAD_SCHEMA_FILE_INFO;
+struct t_mulitload_schema_file_info
+{
+  char schema_file_name[100];
```

Comment:
100 may not be enough.
`char schema_file_name[PATH_MAX];`

Reply:
thanks for your comment.
I'll change it.